### DISCUSSION TITLE
Changing Animations Mid-Dialogue

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#403] [Q&A]
How would I make it so that this could work

```xml
<line char="hopps" changeDefAnim="talk">
		No, no, no... This time's a bit different.
	</line>
	<line char="squig" changeDefAnim="happy">
		So be it! I can still listen to the fun background music!
	</line>
	<line char="squig" changeDefAnim="confused">
		I do still get to listen to the fun background music, right...?
	</line>
	<line char="hopps" changeDefAnim="proud">
		You now EXCLUSIVELY listen to the fun background music!
	</line>
	<line char="squig" changeDefAnim="happy">
		YAAAAAAAAA changeAnim="lessenthusiastic" aaaaaa changeAnim="alexmeyers" what????
	</line>
```

**Comment by r6915ee** [comment]
I'm guessing there might be something within the documentation/API that defines what sort of format to use, I'm just not sure _where_ it could be.

**Reply by THEkatinamicrowave** [reply]
I found this is the api I'ma try see what I can do with this
![image](https://github.com/user-attachments/assets/f800f3f0-6f3b-435c-bc2e-a3464ccd7c8f)

**Comment by NexIsDumb** [comment]
welp if you wanna keep it really simple without too many sweats make a script and check when advancing with the dialogue through events, if the line is the one you wanted place a timer and then change the character's animation
